README.md

```diff
@@ -234,7 +238,7 @@ Go to Transifex, go to the resource, and go to the string. The warning/error mes
 
 ![Transifex screen grab](https://raw.githubusercontent.com/OpenDroneMap/docs/publish/source/readme-img/reST_syntax_err_transifex.png)
 
-Pull the language down (e.g. `tx pull -l sw`) and then try the build again. Unfortunately, if there is more than one error, you'll have to fix them one at a time.
+Pull the language down (for example, `tx pull -l sw`) and then try the build again.
```

Comment:
I suppose an abbreviation of a _Latin_ phrase isn't the best idea in communicating on translating docs in English.